hackathon_site/registration/forms.py

```diff
@@ -149,6 +152,18 @@ def clean(self):
             )
         return cleaned_data
 
+    def clean_birthday(self):
+        latest_birthday = (
+            settings.EVENT_START_DATE - timedelta(days=settings.MINIMUM_AGE * 365)
+        ).date()
+        user_birthday = self.cleaned_data["birthday"]
+        if user_birthday > latest_birthday:
+            raise forms.ValidationError(
+                _("User is too young to participate."),
```

Comment:
We give this error directly to the user, so I would change it to `f"You must be {settings.MINIMUM_AGE} to participate."`. The code is fine though